code/modules/xenoarcheaology/artifacts/artifact.dm

```diff
@@ -5,25 +5,39 @@
 	icon_state = "ano00"
 	var/icon_num = 0
 	density = 1
+
+	var/predefined_icon_num
+
+	var/datum/component/artifact_master/artifact_master = /datum/component/artifact_master
+
+	var/being_used = 0
+
+/*
 	var/datum/artifact_effect/my_effect
```

Comment:
More commented code being kept around - generally not advisable for housekeeping/searching purposes. The logic is in the git history if someone wants to go digging for the old implementation.